test/object_store/test_basic.py

```diff
@@ -119,35 +119,28 @@ async def test_basic(test_tempdir, s3_server, ssl):
         res = conn.execute(f"SELECT * FROM {ks}.{cf};")
 
         r = requests.post(f'http://{ip}:10000/storage_service/keyspace_flush/{ks}', timeout=60)
-        if r.status_code != 200:
-            print(f'Error flushing keyspace: {r}')
-            success = False
+        assert r.status_code == 200, f"Error flushing keyspace: {r}"
 
         # Check that the ownership table is populated properly
         res = conn.execute("SELECT * FROM system.sstables;")
         for row in res:
-            if not row.location.startswith(test_tempdir):
-                print(f'Unexpected entry location in registry: {row.location}')
-                success = False
-            if row.status != 'sealed':
-                print(f'Unexpected entry status in registry: {row.status}')
-                success = False
+            assert row.location.startswith(test_tempdir), \
+                f'Unexpected entry location in registry: {row.location}'
+            assert row.status == 'sealed', f'Unexpected entry status in registry: {row.status}'
```

Comment:
It's not assertion, it's what test actually _tests_, so it should rather be `pytest.fail()`

Reply:
OK, but in the end of this new file there's `pytest.fail()` call. For me it's better to be consistent -- all `pytest.fail()`s or all `assert`s. Or is there a reason for the difference?

Reply:
because 

1. i wanted to preserve the existing behavior, which prints all entries returned by `SELECT * FROM system.sstables;`. and 
2. i was too lazy to refactor it into a single `assert` statement. 

i will restructure it to something like
```py
res = conn.execute("SELECT * FROM system.sstables;")
rows = ";".join(f"{row.location} {row.status}" for row in res)
assert !res, 'Unexpected entry status in registry'
```